src/models/userDao.js

```diff
@@ -0,0 +1,155 @@
+const { AppDataSource } = require('./dataSource');
+
+const findById = async (id) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      id = ?
+    `,
+    [id]
+  );
+
+  return user;
+};
+
+const findBySellerId = async (sellerId) => {
+  const [user] = await AppDataSource.query(
+    `
+      SELECT
+        *
+      FROM
+        users
+      WHERE
+        seller_id = ?
+    `,
+    [sellerId]
+  );
+
+  return user;
+};
```

Comment:
```suggestion
const findUserBySellerId = async (sellerId) => {
  const [user] = await AppDataSource.query(
    `
      SELECT
        *
      FROM
        users
      WHERE
        seller_id = ?
      LMIT 1
    `,
    [sellerId]
  );

  return user;
};
```
- 데이터를 하나 찾으면 탐색을 멈출 수 있도록 제한을 거는 것이 좋겠습니다.
- seller_id가 unique 로 설정되어 있으면 괜찮은데, 아마 아니었던 것 같아서 리뷰드려요

Reply:
넵 알겠습니다!